README.md

```diff
@@ -73,6 +73,14 @@ $ make html
 The docs will be available under `_build/html/index.html`.
 
 
+#### Install the Python extension for Zenoh-Flow
+
+In order to install the Python extension, please execture the following steps:
+- Copy the result of the build into `/var/zenoh-flow/python/`: `sudo cp ./target/release/libzenoh_flow_python* /var/zenoh-flow/python/`
+- Udpdate the path in `01-python.zfext` according to your system. (I.e., replace `.so` with `.dylib` for macOS).
```

Comment:
Update the path in `01-python.zfext` according to your operating system (i.e., replace `.so` with `.dylib` for MacOS).

Reply:
Pretty sure it is called macOS, but I can update it